tests/functional/adapter/basic/test_docs_generate.py

```diff
@@ -43,6 +48,22 @@
 select * from {{ ref('seed') }}
 """
 
+ref_sources__schema_yml = """
+version: 2
+sources:
+  - name: my_source
+    description: "{{ doc('source_info') }}"
+    loader: a_loader
```

Comment:
For the future reviewers - loader is just a description of EL part of ELT. Description only.